packages/commons/src/main/util/ArrayUtil.test.node.js

```diff
@@ -21,6 +21,9 @@
 
 const {ArrayUtil} = require('@wireapp/commons');
 
+/**
+ * @testEnvironment node
```

Comment:
Even if `jsdom` is specified as `testEnvironment` in `jest.config.js` it can be overridden with this comment.

Reply:
https://jestjs.io/docs/en/configuration#testenvironment-string